testing/acceptance_test.py

```diff
@@ -738,8 +728,8 @@ def join_pythonpath(*dirs):
         assert result.ret == 0
         result.stdout.fnmatch_lines(
             [
-                "*lib/foo/bar/test_bar.py::test_bar*PASSED*",
-                "*lib/foo/bar/test_bar.py::test_other*PASSED*",
+                "*lib/foo/bar/test_bar.py::test_bar PASSED*",
+                "*lib/foo/bar/test_bar.py::test_other PASSED*",
```

Comment:
With #4108 this will change to:
```
lib/foo/bar/test_bar.py::test_bar <- local/lib/foo/bar/test_bar.py PASSED [ 50%]
lib/foo/bar/test_bar.py::test_other <- local/lib/foo/bar/test_bar.py PASSED [100%]
```